readme.txt

```diff
@@ -4,7 +4,7 @@
 
 = [4.5.10] TBD =
 
-
+* Fix - Added support to tribe_asset() for altering plugin dir or using WPMU plugins. Thanks to @squirrelandnnuts for reporting this [82809]
```

Comment:
More props to you for giving me props for my acknowledgement of the props you gave to squirrelandnuts